README.md

```diff
@@ -0,0 +1,13 @@
+Codefellows 201 Guessing Game
+
+files:
+  index.html
+  app.js
+  README.md
+
+citations:
+  https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/toLowerCase
+
+author:
+  Geoffrey Cox
+  gcox3d@gmail.com
```

Comment:
Good README! Try using headers and bullet points. If you need help, let me know.